src/test/java/com/jcabi/xml/XMLDocumentTest.java

```diff
@@ -349,9 +349,10 @@ public void buildsDomNode() throws Exception {
      */
     @Test
     public void comparesToAnotherDocument() throws Exception {
+        final String xml = "<hi><dude>  </dude></hi>";
         MatcherAssert.assertThat(
-            new XMLDocument("<hi>\n<dude>  </dude></hi>"),
-            Matchers.equalTo(new XMLDocument("<hi><dude>  </dude></hi>"))
+            new XMLDocument(xml),
```

Comment:
It is confusing to have unrelated asserts in one test method, please split those into separate tests.